pcsx2/ImGui/ImGuiManager.cpp

```diff
@@ -780,10 +780,10 @@ void ImGuiManager::DrawOSDMessages(Common::Timer::Value current_time)
 		if (actual_y >= s_window_height)
 			break;
 
-		const ImVec2 pos(position_x, actual_y);
 		const ImVec2 text_size(
 			font->CalcTextSizeA(font->FontSize, max_width, max_width, msg.text.c_str(), msg.text.c_str() + msg.text.length()));
 		const ImVec2 size(text_size.x + padding * 2.0f, text_size.y + padding * 2.0f);
+		const ImVec2 pos(position_x - (size.x * (GSConfig.OsdMessagesPos == OsdOverlayPos::TopRight)), actual_y);
```

Comment:
I'd perfer to make it clearer with a ternary instead of mixing boolean logic and numbers via the multiply
```suggestion
		const ImVec2 pos(position_x - ((GSConfig.OsdMessagesPos == OsdOverlayPos::TopRight) ? size.x : 0), actual_y);
```

Reply:
Agreed; done.